packages/ui/src/components/footer/footer.vue

```diff
@@ -1,15 +1,13 @@
 <script lang="ts" setup>
-  import type { AcvFooterSlots } from './footer.ts';
   import AcvDivider from '../divider/divider.vue';
 
   defineOptions({ name: 'AcvFooter' });
-  defineSlots<AcvFooterSlots>();
```

Comment:
Why are you removed slots? We provide slot types here

Reply:
It's the way we do it for button component. I think this decision was made due to simplification

Reply:
But we will loose docs for Footer slots at least. 
We use vue-docgen to collect available data from sources and generate md files

Reply:
Try to regenerate docs for the footer, if it will keep the slots them you can do that

Reply:
Now it works, even without footer.ts, because vue docgen gets info from component file directly and automatically assigns slots with their descriptions. I just used comments above slots where we want descriptions and it works, this is how it was done in button as well

Reply:
I tested in button and `vue-docgen` gives slots from the template